extras/pre-commit-hooks-extra.nix

```diff
@@ -15,6 +16,27 @@
             entry = "${rustfmt}/bin/rustfmt --color always";
             files = "\\.rs$";
           };
+
+
+        # TODO(jaredponn): I'm fairly certain later versions of flake-parts
+        # includes `denofmt` and `denolint` so we don't have to reimplement
+        # this ourselves
```

Comment:
You mean pre-commit-hooks.nix? `nix flake lock --update-input <>` whatever needs to be updated.